Describe how to import old hash onion in Lisk docs (#1841)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/run-blockchain/enable-block-generation.adoc

----
. xref:{url_run_validator_import}[Import the validator keys].
. xref:{url_run_validator_set_hashonion}[Import the hash onion seed] used by the validator.

> . Import the old hash onion to the new node.
> +
> [source,bash]
> ----
> lisk-core endpoint:invoke random_setHashOnion '{"address":"lskz45myuptyyjvzc2z7o5344quubyrv6nofffh8e","seed":"2aa20ce749838a5c6bac8bdf14dcebd8","count":1000000,"distance":1000}'
> ----
> 

Reviewer: This is redundant as we already directed the user to import the hash onions in the preceding point. Having this point here will mean that the user has to redo the process again. I have added a comment above to add more information in the setting hash onion section.
```suggestion
```

> . Ensure the node is fully synchronized with the network. This process ensures that the new node is up-to-date with the latest blocks and transactions in the network.
> Hence, the height of your node should be equal to the current network height.

Reviewer: ```suggestion
. Ensure the node is fully synchronized with the network.
Synchronization ensures that the new node is up-to-date and has the latest blocks and transactions from the network.
After sync, the height of the newer node should be equal to the current network height.
```
